commands/project/add.ts

```diff
@@ -23,40 +29,91 @@ exports.describe = uiBetaTag(i18n(`${i18nKey}.describe`), false);
 exports.handler = async options => {
   const { derivedAccountId } = options;
 
-  logger.log('');
-  logger.log(i18n(`${i18nKey}.creatingComponent.message`));
-  logger.log('');
+  trackCommandUsage('project-add', null, derivedAccountId);
 
-  const releaseData = await fetchReleaseData(
-    HUBSPOT_PROJECT_COMPONENTS_GITHUB_PATH
-  );
-  const projectComponentsVersion = releaseData.tag_name;
+  const { projectConfig, projectDir } = await getProjectConfig();
 
-  const components = await getProjectComponentsByVersion(
-    projectComponentsVersion
+  if (!projectDir || !projectConfig) {
+    logger.error(i18n(`${i18nKey}.error.locationInProject`));
+    process.exit(EXIT_CODES.ERROR);
+  }
+
+  // We currently only support adding private apps to projects
+  let projectContainsPublicApp = false;
```

Comment:
This is the check to determine whether or not we're in a project that contains a public app. I re-used the `findProjectComponents` util that we use for local dev